results/migrations/0018_add_partners_2012_data.py

```diff
@@ -0,0 +1,198 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+import os
+
+import json
+from itertools import chain
+
+from django.core.files import File
+from django.db import migrations
+from django.utils.text import slugify
+
+from cpm_generic.migration_utils import (add_subpage, get_content_type,
+                                         get_image_model, remove_subpage)
+
+MIGRATION_DIR = os.path.dirname(__file__)
+DIR_0017 = os.path.join(MIGRATION_DIR, '0018_add_partners_2012_data')
+
+
+# TODO: coulds be abstracted
```

Comment:
Why data is not in https://github.com/kinaklub/next.filmfest.by/blob/master/data/raw/cpm2012/partners.json ?

    from helpers import get_data

    get_data('cpm2012/partners.json')